src/xo-app/dashboard/health/index.js

```diff
@@ -319,12 +320,12 @@ const ALARM_COLUMNS = [
     .sort()
   const getControlDomainVdis = createSelector(
     getControlDomainVbds,
-    createGetObjectsOfType('VDI'),
     createGetObjectsOfType('pool'),
     createGetObjectsOfType('SR'),
-    (vbds, vdis, pools, srs) =>
+    state => state,
+    (vbds, pools, srs, state) =>
       mapPlus(vbds, (vbd, push) => {
-        const vdi = find(vdis, { id: vbd.VDI })
+        const vdi = getObject(state, vbd.VDI)
```

Comment:
Ok, let's keep using `createGetObjectsOfType('VDI)`, but it returns a map, so no need to use `find`.